project/plugins.sbt

```diff
@@ -15,10 +15,7 @@ addSbtPlugin("pl.project13.scala" % "sbt-jmh" % "0.3.7")
 addSbtPlugin("com.typesafe.sbt" % "sbt-native-packager" % "1.4.1")
 addSbtPlugin("io.spray" % "sbt-boilerplate" % "0.6.1")
 addSbtPlugin("com.timushev.sbt" % "sbt-updates" % "0.4.2")
-addSbtPlugin("com.lightbend.akka" % "sbt-paradox-akka" % "0.24")
-addSbtPlugin("com.lightbend.paradox" % "sbt-paradox-apidoc" % "0.3")
-addSbtPlugin("com.lightbend.paradox" % "sbt-paradox-project-info" % "1.1.3")
-addSbtPlugin("com.lightbend.paradox" % "sbt-paradox" % "0.6.5")
+addSbtPlugin("com.lightbend.akka" % "sbt-paradox-akka" % "0.25")
```

Comment:
👍

Reply:
ok, so this means that whenever there is a release of apidoc and project-info there will be a new release of paradox-akka.